main.cc

```diff
@@ -1894,8 +1894,10 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 view_backlog_broker.stop().get();
             });
 
-            startlog.info("Waiting for gossip to settle before accepting client requests...");
-            gossiper.local().wait_for_gossip_to_settle().get();
+            if (!ss.local().raft_topology_change_enabled()) {
+                startlog.info("Waiting for gossip to settle before accepting client requests...");
+                gossiper.local().wait_for_gossip_to_settle().get();
```

Comment:
What about the other waits?

Reply:
Added skip for the second `wait_for_gossip_to_settle`